tutoraspects/asset_command_helpers.py

```diff
@@ -243,12 +263,8 @@ def validate_asset_file(asset_path, content, echo):
 
             existing = None
 
-            # Check if the file already exists
-            if os.path.exists(os.path.join(out_path, out_filename)):
-                with open(
-                    os.path.join(out_path, out_filename), encoding="utf-8"
-                ) as stream:
-                    existing = yaml.safe_load(stream)
+            existing = _check_file_exists(out_path, out_filename_trimmed)
+            existing = _check_file_exists(out_path, out_filename_uuid)
```

Comment:
I think we should only be doing one of these, and can maybe use `content[FILE_NAME_ATTRIBUTE]` for it?

Reply:
I had it checking both for the initial renaming because it needed to find the original file to do the replacement but we probably don't need it anymore if we're going to rename everything all at once. I'll change it!